service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/ReturnedCaseContentProvider.java

```diff
@@ -36,7 +36,7 @@ private ReturnedCaseTemplate.ReturnedCaseTemplateBuilder templateData(CaseData c
         ReturnApplication returnApplication = caseData.getReturnApplication();
 
         return ReturnedCaseTemplate.builder()
-            .localAuthority(laLookup.getLocalAuthorityName(caseData.getCaseLocalAuthority()))
+            .localAuthority(laLookup.getLocalAuthorityName(caseData.getCaseLaOrRelatingLa()))
```

Comment:
I think looking at the template - it looks like this field should be the applicant name rather than the LA name? (thinking for 3rd party standalones, it wouldn't be relatingLA - think this came up once before in a similar ticket, might be a util function to grab the applicant name?